gamecenter/api/views.py

```diff
@@ -55,12 +56,48 @@ def signup():
 
 
 @blueprint.route('/leaderboards', methods=['GET', 'POST'])
+@handle_api_key
 @get_request_args
 def leaderboards_controller(args):
     if request.method == 'GET':
         return get_paginated_scores(args)
     else:
-        return create_entry()
+        return jsonify(data=SCORESCHEMA.dump(create_entry()).data)
+
+
+@blueprint.route('/add_score', methods=['POST'])
```

Comment:
okay, then let's make a task to modify the paper to take out anything that says our service is RESTful